core/src/main/resources/hudson/model/View/sidepanel.jelly

```diff
@@ -33,6 +33,10 @@ THE SOFTWARE.
     <link rel="alternate" title="Jenkins:${it.viewName} (all builds) (RSS 2.0)" href="${rootURL}/${it.url}rssAll?flavor=rss20" type="application/rss+xml" />
     <link rel="alternate" title="Jenkins:${it.viewName} (failed builds)" href="${rootURL}/${it.url}rssFailed" type="application/rss+xml" />
     <link rel="alternate" title="Jenkins:${it.viewName} (failed builds) (RSS 2.0)" href="${rootURL}/${it.url}rssFailed?flavor=rss20" type="application/rss+xml" />
+
+    <l:css src="jsbundles/remodal.css" />
```

Comment:
I removed new styles from style.css and added remodal library styles to remodal.css
Is this correct?